dump/mapping/bufr_satwnd_ascat.py

```diff
@@ -0,0 +1,125 @@
+                dummy = container.get('windSpeedAt10M', cat)
+                container.add('windEastward', dummy, paths, cat)
+                container.add('windNorthward', dummy, paths, cat)
+
+                paths = container.get_paths('satelliteId', cat)
+                dummy = container.get('satelliteId', cat)
+                container.add('obstype_windEastward', dummy, paths, cat)
+                container.add('obstype_windNorthward', dummy, paths, cat)
+                continue
+
+            wdir = container.get('windDirectionAt10M', cat)
+            wspd = container.get('windSpeedAt10M', cat)
+            self.log.debug(f'wdir min/max = {wdir.min()} {wdir.max()}')
+            self.log.debug(f'wspd min/max = {wspd.min()} {wspd.max()}')
+
+            uob, vob = self._wind_helper._compute_wind_components(wdir, wspd)
+            self.log.debug(f'uob min/max = {uob.min()} {uob.max()}')
+            self.log.debug(f'vob min/max = {vob.min()} {vob.max()}')
+
+            paths = container.get_paths('windSpeedAt10M', cat)
+            container.add('windEastward', uob, paths, cat)
+            container.add('windNorthward', vob, paths, cat)
+
+            obstype = self._make_obs_type(container, cat)
+            paths = container.get_paths('satelliteId', cat)
+            container.add('obstype_windEastward', obstype, paths, cat)
+            container.add('obstype_windNorthward', obstype, paths, cat)
+
+        # Check
+        self.log.debug(f'container list (updated): {container.list()}')
+        self.log.debug(f'all_sub_categories {container.all_sub_categories()}')
+
+        return container
+
+    def _make_obs_type(self, container, category):
+        satId = container.get('satelliteId', category)
+
+        if not satId.size:
+            paths = container.get_paths('satelliteId', cat)
+            dummy = container.get('satelliteId', cat)
+            container.add('obstype_windEastward', dummy, paths, cat)
+            container.add('obstype_windNorthward', dummy, paths, cat)
+            return
+
+        obstype = np.full_like(satId, 290)
+
+        return obstype
+
+    def _make_description(self):
+        description = super()._make_description()
+        self._add_wind_components_descriptions(description)
+        self._add_obs_type_descriptions(description)
+
+        return description
+
+    def _add_wind_components_descriptions(self, description):
+        description.add_variables([
+            {
+                'name': 'ObsValue/windEastward',
+                'source': 'windEastward',
+                'units': 'm s-1',
+                'longName': '10-meter U-Wind Component',
+            },
+            {
+                'name': 'ObsValue/windNorthward',
+                'source': 'windNorthward',
+                'units': 'm s-1',
+                'longName': '10-meter V-Wind Component',
+            }])
+
+    def _add_obs_type_descriptions(self, description):
```

Comment:
Can this be merged with ```_add_wind_components_descriptions``` above?

Reply:
Good idea.
I merged the two descriptions together.

Reply:
Thanks!